tests/test_keystone.py

```diff
@@ -0,0 +1,35 @@
+import os
+import logging
+import nose
+import angr
+
+l = logging.getLogger("angr.tests")
+test_location = str(os.path.join(os.path.dirname(os.path.realpath(__file__)), '../../binaries/tests'))
+
+target_addrs = {
+    'i386': [ 0x080485C9 ],
+    'x86_64': [ 0x4006ed ],
+    'armel': [ 0x85F0 ],
+}
+
+def run_keystone(arch):
+    p = angr.Project(os.path.join(test_location, arch, "fauxware"))
+
+    block = p.factory.block(target_addrs[arch][0])
+    sm = p.factory.simgr()
```

Comment:
I think there was a consensus among the dev team that `factory.simgr` isn't super understandable, so we're using the longform `factory.simulation_manager` now